React/Views/RCTBorderDrawing.m

```diff
@@ -484,8 +485,6 @@ static CGContextRef RCTUIGraphicsBeginImageContext(CGSize size, CGColorRef backg
   CGContextSetLineWidth(ctx, lineWidth);
   CGContextSetLineDash(ctx, 0, dashLengths, sizeof(dashLengths) / sizeof(*dashLengths));
 
-  CGContextSetStrokeColorWithColor(ctx, [UIColor yellowColor].CGColor);
```

Comment:
I took this out because it looks like it was there by mistake. The stroke color gets overwritten two lines down, before the next draw operation.